packages/cli-doctor/src/tools/healthchecks/ruby.ts

```diff
@@ -0,0 +1,24 @@
+import versionRanges from '../versionRanges';
+import {doesSoftwareNeedToBeFixed} from '../checkInstallation';
+import {HealthCheckInterface} from '../../types';
+
+export default {
+  label: 'Ruby',
+  isRequired: false,
+  getDiagnostics: async ({Managers}) => ({
+    needsToBeFixed: doesSoftwareNeedToBeFixed({
+      version: Managers.RubyGems.version,
+      versionRange: versionRanges.RUBY,
+    }),
+    version: Managers.RubyGems.version,
+    versionRange: versionRanges.RUBY,
+  }),
+  runAutomaticFix: async ({loader, logManualInstallation}) => {
+    loader.fail();
+
+    logManualInstallation({
+      healthcheck: 'Ruby',
+      url: 'https://www.ruby-lang.org/en/downloads/',
```

Comment:
WDYT about using [this link ](https://reactnative.dev/docs/environment-setup#ruby) instead? :)

Reply:
Was relying on download link for Ruby considering Node example :
https://github.com/react-native-community/cli/blob/main/packages/cli-doctor/src/tools/healthchecks/nodeJS.ts